src/app/work-item/work-item-detail/work-item-detail.component.html

```diff
@@ -196,70 +196,85 @@ <h2 *ngIf="loggedIn && workItem.id" id="wi-detail-title-click" class="detail-tit
           <label class="col-sm-2" id="area_label">
             {{workItem.relationships?.baseType?.data?.attributes?.fields['system.area']?.label}}
           </label>
-          <div class="col-sm-10 detail-area-name pull-left">
-              <div *ngIf="!searchArea" (click)="activeSearchArea()"
-                class="pointer" id="WI_details_area">
-                {{ workItem.relationships?.area?.data?.attributes?.name || "None" }}
-              </div>
-              <div style="display: flex;">
-                <div *ngIf="searchArea" class="area-search">
-                  <div class="list-container">
-                    <span class="fa fa-search area-search-icon"> </span>
-                    <input id="areaSearchInput" #areaSearch (keyup)="filterArea($event)"
-                      autocomplete="off" />
-                    <ul class="item-ul area-list" #areaList>
-                      <li *ngIf="!areas.length" class="textCenter"><i class="fa fa-spinner fa-pulse"></i></li>
-                      <li
-                        class="item-li"
-                        *ngFor="let area of filteredAreas;"
-                        [attr.data-value]="area.id"
-                        (click)="showAreaOnInput(area)" [id]="area.id">
-                          <span [innerHTML]="area.attributes?.name | almSearchHighlight: areaSearch.value"></span>
-                      </li>
-                    </ul>
-                  </div>
-                </div>
-                <div *ngIf="searchArea" class="btn-bar"  >
-                  <button class="btn btn-lg btn-default"  type="button" (click)=setArea() id="save_area">
-                    <span class="fa fa-check" ></span>
-                  </button>
-                  <button class="btn btn-lg btn-default"  type="button" (click)=closeAreaDropdown()>
-                    <span class="pficon pficon-close" id="close_area" ></span>
-                  </button>
+          <div class="combobox-container col-sm-6 padding-0 detail-area-name">
+            <input type="hidden" name="" value="">
+            <span *ngIf="!searchArea" (click)="activeSearchArea()"
+               [class.none-assigned]="!workItem.relationships?.area?.data?.attributes?.name" class="pointer" id="WI_details_area">
+               {{ workItem.relationships?.area?.data?.attributes?.name || "None" }}
+            </span>
+            <div class="flex-container">
+              <div *ngIf="searchArea" class=" col-sm-6 padding-0">
+                <div class="input-group " dropdown>
+                  <input id="areaSearchInput" #areaSearch (keyup)="filterArea($event)" placeholder="Select Area"
+                    autocomplete="off" value="{{ workItem.relationships?.area?.data?.attributes?.name }}"
```

Comment:
To make area selection functional
1. Remove value="{{ workItem.relationships?.area?.data?.attributes?.name }}"